wazo_calld/plugin_helpers/confd.py

```diff
@@ -164,6 +164,16 @@ def interface_autoanswer(self):
             interface = f'{interface}/autoanswer'
         return interface
 
+    def is_online(self, ari):
+        line = self._get()
+        name = line['name']
+        tech = line['protocol']
+        if tech == 'sip':
+            return (
+                ari.endpoints.get(tech='pjsip', resource=name).json['state'] == 'online'
+            )
```

Comment:
according to the ticket acceptance criteria, sccp should be supported too.

Also, I would log something here if the tech is not supported to explain that this `True` is not actually a true positive. 

Alternatively, don't limit the "tech" here, but move that logic in the layer above(where that method is called). IMHO, usually a better design to keep special case logic/non technical concerns in higher layers(higher level business logic, or near system boundaries like http handlers or service classes) , and keep utility code simpler.

Reply:
I ended up adding it there to avoid duplicating the verification, but yeah, I can move it to the layer above

Reply:
It has been moved :+1: